docs/dataformats/observation_lists.rst

```diff
@@ -10,38 +10,87 @@ the information in the run list. These filenames can either be given directly
+TIME_STOP         end time of the observation in the same forma as TIME_START                                       yes?
+TIME_OBSERVATION  duration of the observation                                                                       yes?
+TIME_LIVE         duration of the observation without dead time                                                     yes?
+TRIGGER_RATE      average trigger rate of the system                                                                no
+MEAN_TEMPERATURE  average temperature of the atmosphere                                                             no
+N_TELS            number of telescopes participating in the observation                                             yes?
+TEL_LIST          string with a CSV list of telescope IDs participating in the observation                          yes?
+QUALITY           data quality; recommended: "spectral" or "detection" (not used by Gammapy at the moment)          no
+================  ================================================================================================  =========
 
-* `LibreOffice Calc <http://www.libreoffice.org/discover/calc/>`_ 
-* `TOPCAT <http://www.star.bris.ac.uk/~mbt/topcat/>`_ or `STILTS <http://www.star.bris.ac.uk/~mbt/stilts/>`_
-* `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
-* Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
+Extra user defined columns are allowed; Gammapy will ignore them.
 
-A run list must have at least a column called ``Run``::
- 
-   Run
-   42
-   43
+In order for the extra columns to have full meaning the following is needed:
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+ * Extra row right after the column name, specifying the unit of the quantity listed on each column.
+ * A header with at least the following keywords:
+
+================  ==========================================================================================================================================================================================================
+keyword           description
+================  ==========================================================================================================================================================================================================
+OBSERVATORY_NAME  name of the observatory where the observations were taken. This is important for instance for coordinate transformations between celestial (i.e. RA/dec) and terrestrial (i.e. az/alt) coordinate systems.
+MJDREFI           reference time for other times in the list (i.e. TIME_START/TIME_STOP). Integer value in mean julian days.
```

Comment:
For `MET` and the reference time it really would be best to have a single place where this is explained in Gammapy.
Then link to it from here and from the event list format specification and class, instead of trying to quickly re-explain this in several places.